modules/shared/src/main/scala/org/tessellation/currency/utils/DataTransactionUtils.scala

```diff
@@ -0,0 +1,70 @@
+package org.tessellation.currency.utils
+
+import cats.data.NonEmptyList
+import cats.effect.Async
+import cats.syntax.all._
+
+import org.tessellation.currency.dataApplication.FeeTransaction.serializeFeeTransaction
+import org.tessellation.currency.dataApplication.{DataTransaction, DataUpdate, FeeTransaction}
+import org.tessellation.security.hash.Hash
+import org.tessellation.security.signature.Signed
+import org.tessellation.security.{Hasher, hash}
+
+object DataTransactionUtils {
```

Comment:
Please don't create `Utils` files. It's a bag for everything and it will grow and grow. I'm sure we can place those functions somewhere next to its usage, name them properly, etc. Get data/fee may be in the companion object of `DataTransaction`, named `collectDataUpdates` and `collectFeeTransactions`, or `DataUpdate.collect(dataTransactions)` and `FeeTransactions.collect(dataTransactions)`.

Reply:
Got it, my intention was to not increase too much the package file adding this to the companion objects

Reply:
If so, maybe it's worth moving it from the package file to `DataTransaction.scala` or sth?